dask_expr/_expr.py

```diff
@@ -1202,6 +1198,14 @@ def _simplify_up(self, parent):
                 type(self)(self.frame[sorted(columns)], *self.operands[1:]),
                 *parent.operands[1:],
             )
+        elif isinstance(parent, Projection):
```

Comment:
We can't do this. We drop rows if any column contains a missing value. The operation will change if we remove columns that could potentially contain NA values

Reply:
Ah, Good catch! I don't know what I was thinking :)